notes/stories/literature/novel-various.st.txt

```diff
@@ -1,3 +1,73 @@
+novel by Jules Verne. It was first published in French in 1864, then reissued
+in 1867 in a revised and expanded edition. Professor Otto Lidenbrock is the
+tale's central figure, an eccentric German scientist who believes there are
+volcanic tubes that reach to the very center of the earth. He, his nephew
+Axel, and their Icelandic guide Hans rappel into Iceland's celebrated inactive
+volcano Snæfellsjökull, then contend with many dangers, including cave-ins,
+subpolar tornadoes, an underground ocean, and living prehistoric creatures
+from the Mesozoic and Cenozoic eras. (The 1867 revised edition inserted
+additional prehistoric material in Chaps. 37–39.) Eventually the three
+explorers are spewed back to the surface by an active volcano, Stromboli,
+in southern Italy.
+
+:: Authors
+Jules Verne
+
+:: References
+https://en.wikipedia.org/wiki/Journey_to_the_Center_of_the_Earth
+
+:: Ratings
+5 <paul>
+
+:: Choice Themes
+travel into the Earth [The whole story is about a trio of intrepid explorers who trek down old lava tubes to ultimately reach the center of the Earth.]
+
+:: Major Themes
+the thirst for knowledge [Professor Lidenbrock was driven by a desire to discover what lurked deep beneath the surface of the Earth. On the journey down, he comported himself in a single-minded and at times reckless manner so that he might satisfy his burning curiosity.]
+uncle and nephew [Professor Lidenbrock and his nephew Alex explored the bowels of the Earth together.]
+human vs. nature [Professor Lidenbrock, his nephew Alex, and their stoic Icelandic guide Hans contended with all types of dangers as they trekked down through an extinct volcano to the center of the Earth: the survived cave-ins, privation, tornadoes, a perilous voyage on a vast subsurface ocean, an encounter with a prehistoric sea creature, etc.]
+Hollow Earth [Professor Lidenbrock and his party pointedly proved wrong the prevailing theory that the Earth had a central source of intense heat by the simple expedient of trekking down the lava tubes of an extinct volcano all the way to the center of the planet.]
+geology [Professor Lidenbrock and his party commented upon numerous geological features on their trek down to the center of the Earth: Lidenbrock (and to a lesser extent Alex) relatively dated the geological strata they were passing through and examined their fossil contents, discussed about various minerals, entertained two opposing theories of what lay beneath the Earth, etc.]
+academia [Professor Lidenbrock was a stereotypical oblivious German academic who was interested solely in advancing science by disproving the prevailing theory that the Earth had an intense source of heat at its center.]
+the scientist profession [The trained geologist Professor Lidenbrock and his nephew protégé Alex were in their element as they explored the bowels of the Earth.]
+human character stereotype [The party guide Hans was portrayed as a stoic Scandinavian to the extreme.]
```

Comment:
I approve.

Reply:
Hah

Reply:
To be honest I think this stereotype is too vague to identify consistently. But possibly being of this noble lineage myself I just don't realise how imposing we may seem to lesser mortals.

Reply:
o_0

Reply:
How about using the general **human character** theme here? We used to have **stoicism** in there, but as things stand there's nothing to cover his nature of which much was made.

Reply:
Perfect!